regression/contracts/function_check_02/main.c

```diff
@@ -0,0 +1,23 @@
+// function_check_02
+
+// This test checks the use of quantifiers in ensures clauses.
+// A known bug causes the use of quantifiers in ensures to fail.
```

Comment:
I believe this is #2278?

Reply:
Yes, it is. Should the comment be updated to account for the existence of that PR?